database/models/thermo_transport.py

```diff
@@ -48,77 +49,72 @@ class Thermo(models.Model):
     coeffs_poly2 = ArrayField(models.FloatField(), size=7)
     temp_min_1 = models.FloatField("Polynomial 1 Lower Temp Bound", help_text="units: K")
     temp_max_1 = models.FloatField("Polynomial 1 Upper Temp Bound", help_text="units: K")
+    # shouldn't temp_min_2 always equal temp_max_1, and we just store T_min, T_middle, T_max ?
```

Comment:
This was also written before I got here. Looking at the RMG `NASA` object though, that makes sense, though there is no middle temperature. Is that just the average of the min and max?